_layouts/base.html

```diff
@@ -12,10 +12,19 @@
       name="viewport"
       content="width=device-width, initial-scale=1, user-scalable=no"
     />
-    <meta property=”og:title” content="{{ page.title | default: site.title }}" />
-    <meta property=”og:url” content="https://codevid19.com" />
-    <meta property=”og:image” content="https://codevid19.com/images/codevid-19_insta.png" />
-    <meta property="og:description" content="{{ page.description | default: site.description }}" />
+    <meta
+      property="”og:title”"
+      content="{{ page.title | default: site.title }}"
+    />
+    <meta property="”og:url”" content="https://codevid19.com" />
+    <meta
+      property="”og:image”"
+      content="https://codevid19.com/images/codevid-19_insta.png"
+    />
+    <meta
+      property="og:description"
+      content="{{ page.description | default: site.description }}"
+    />
```

Comment:
Think this is because not all maintainers are using Prettier.  I think it's fine that they appear right now but maybe on the weekend we should look into a CI Action that runs Prettier on our project prior to deployment.

Reply:
Sure, we should also add a Prettier addon to the VS Code dev container, then as long as the settings are there if can do that on file save.